tests/test_spidermiddleware_stickymeta.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from unittest import TestCase
```

Comment:
:lipstick: I would expect this standard library import to be above the third-party `pytest` import.

Reply:
Done :heavy_check_mark: